packages/app-extensions/src/form/syncValidation.js

```diff
@@ -22,25 +22,24 @@ const addErrors = (errors, field, fieldErrors) => (
   }
 )
 
-const validate = (values, entityModel) => {
+export const validate = (values, entityModel) => {
   let errors = {}
 
-  errors = validateModel(values, entityModel, errors)
-  errors = validateTypes(values, entityModel, errors)
+  errors = syncValidateModel(values, entityModel, errors)
+  errors = syncValidateTypes(values, entityModel, errors)
 
   return errors
 }
 
-const validateTypes = (values, entityModel, errors) => {
+export const syncValidateTypes = (values, entityModel, errors) => {
```

Comment:
why export? as far is i see you dont import it anywhere

Reply:
why rename? Since its not exported and the file name indicates the sync part it can be named "validateTypes" IMO